imap_processing/tests/conftest.py

```diff
@@ -18,3 +22,107 @@ def _set_global_config(monkeypatch, tmp_path):
 @pytest.fixture(scope="session")
 def imap_tests_path():
     return imap_module_directory / "tests"
+
+
+# Furnishing fixtures for testing kernels
+# ---------------------------------------
+@pytest.fixture(autouse=True)
+def _autoclear_spice():
+    """Automatically clears out all SPICE remnants after every single test to
+    prevent the kernel pool from interfering with future tests. Option autouse
+    ensures this is run after every test."""
+    yield
+    spice.kclear()
+
+
+@pytest.fixture(scope="session")
+def spice_test_data_path(imap_tests_path):
+    return imap_module_directory.parent / "tools/tests/test_data/spice"
```

Comment:
I _think_ we might want to move these data files into the main package. I'm pretty sure if we released this, the tests would get included in the release, but the test data wouldn't so you wouldn't be able to test the actual sdist because the test data would be missing.